SimplyPluralApi.yml

```diff
@@ -490,96 +406,85 @@ paths:
         - APITokenAuth: []
       summary: "Update custom front"
       description: "Update a Custom Front by Id"
+      parameters:
+        $ref: "./SimplyPluralApiCommons.yml#/components/parameters/objectId"
       requestBody:
         required: true
         content:
           application/json:
             schema:
               $ref: "./SimplyPluralApiBodies.yml#/components/schemas/customFront"
-      parameters:
-        - in: path
-          name: docId
-          schema:
-            type: string
-          required: true
-          description: String Id of the Custom Front
+      tags:
+        - Custom Fronts
       responses:
         "200":
           description: OK
         "404":
           description: NOT FOUND
-      tags:
-        - Custom Fronts
     delete:
       security:
         - APITokenAuth: []
       summary: "Delete custom front"
       description: "Delete a Custom Front by Id"
+      parameters:
+        $ref: "./SimplyPluralApiCommons.yml#/components/parameters/objectId"
+      tags:
+        - Custom Fronts
       responses:
         "200":
           description: OK
         "404":
           description: NOT FOUND
-      tags:
-        - Custom Fronts
+
   /v1/customFronts/{systemId}:
     get:
       summary: "Get all custom fronts"
       description: "Get custom fronts of a system by Id"
+      parameters:
+        $ref: "./SimplyPluralApiCommons.yml#/components/parameters/systemId"
       tags:
         - Custom Fronts
-      parameters:
-        - in: path
-          name: systemId
-          schema:
-            type: string
-          required: true
-          description: String Id of the system
       responses:
         "200":
           description: OK
         "404":
           description: NOT FOUND
 
-  /v1/comment/{type}/{docId}:
+  /v1/comment/{type}/{id}:
     get:
       security:
         - APITokenAuth: []
       summary: "Get comments"
-      description: "Get comments from a document of type (groups, members, customFronts, etc.)"
-      tags:
-        - Comments
+      description: "Get comments from a document of a particular type (groups, members, customFronts, etc.)"
```

Comment:
I am not sure why it says get comments for these types in the current docs, the only type supported is "frontHistory"